.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,87 @@
+# Ultralytics 🚀 AGPL-3.0 License - https://ultralytics.com/license
+
+name: 🐛 Bug Report
+description: "Problems with the Ultralytics Handbook site"
+labels: [bug, triage]
+type: "bug"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting an Ultralytics Handbook 🐛 Bug Report!
+
+  - type: checkboxes
+    attributes:
+      label: Search before asking
+      description: >
+        Please search the Ultralytics Handbook [site](https://handbook.ultralytics.com/) and [issues](https://github.com/ultralytics/handbook/issues) to see if a similar bug report already exists.
+      options:
+        - label: >
+            I have searched https://github.com/ultralytics/handbook/issues and did not find a similar report.
+          required: true
+
+  - type: dropdown
+    attributes:
+      label: Project area
+      description: |
+        Help us route the report to the right maintainers.
+      multiple: true
+      options:
+        - "Content pages"
+        - "Navigation or search"
+        - "Onboarding/people guides"
+        - "Build tooling (Zensical)"
+        - "Deployment (Vercel)"
+        - "Other"
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Bug
+      description: Please describe the issue in detail so we can reproduce it in Ultralytics Handbook. Include logs, screenshots, console output, and any context that helps explain the problem.
+      placeholder: |
+        💡 ProTip! Include as much information as possible (logs, tracebacks, screenshots, etc.) to receive the most helpful response.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Environment
+      description: Share the platform and version information relevant to your report.
+      placeholder: |
+        Please include:
+        - OS (e.g., Ubuntu 20.04, macOS 13.5, Windows 11)
+        - Language or framework version (Python, Swift, Flutter, etc.)
+        - Package or app version
+        - Hardware (e.g., CPU, GPU model, device model)
+        - Any other environment details
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Minimal Reproducible Example
+      description: >
+        Provide the smallest possible snippet, command, or steps required to reproduce the issue. This helps us pinpoint problems faster.
+      placeholder: |
+        ```python
+        # Code or commands to reproduce your issue here
+        ```
+    validations:
+      required: true
+
+  - type: textarea
```

💭 **SUGGESTION**: The `Additional` section is missing explicit `validations`. This is fine (optional by default), but adding `validations` consistently across sections can make the structure clearer to contributors and maintainers.

**Suggested change:**
```suggestion
  - type: textarea
```